src/main/java/at/hannibal2/skyhanni/features/mining/fossilexcavator/ExcavatorProfitTracker.kt

```diff
@@ -0,0 +1,182 @@
+
+    private val config get() = SkyHanniMod.feature.mining.fossilExcavator.profitTracker
+
+    private val tracker = SkyHanniItemTracker(
+        "Fossil Excavation Profit Tracker",
+        { Data() },
+        { it.mining.fossilExcavatorProfitTracker }) { drawDisplay(it) }
+
+    class Data : ItemTrackerData() {
+        override fun resetItems() {
+            timesExcavated = 0
+        }
+
+        override fun getDescription(timesGained: Long): List<String> {
+            val percentage = timesGained.toDouble() / timesExcavated
+            val dropRate = LorenzUtils.formatPercentage(percentage.coerceAtMost(1.0))
+            return listOf(
+                "§7Dropped §e${timesGained.addSeparators()} §7times.",
+                "§7Your drop rate: §c$dropRate."
+            )
+        }
+
+        override fun getCoinName(item: TrackedItem) = "<no coins>"
+
+        override fun getCoinDescription(item: TrackedItem): List<String> {
+            return listOf(
+                "<no coins>"
+            )
+        }
+
+        @Expose
+        var timesExcavated = 0L
+        var glacitePowderGained = 0L
+        var fossilDustGained = 0L
+    }
+
+    private val scrapItem = "SUSPICIOUS_SCRAP".asInternalName()
+
+    private fun drawDisplay(data: Data): List<List<Any>> = buildList {
+        addAsSingletonList("§e§lFossil Excavation Profit Tracker")
+        var profit = tracker.drawItems(data, { true }, this)
+
+        val timesExcavated = data.timesExcavated
+        addAsSingletonList(
+            Renderable.hoverTips(
+                "§7Times excavated: §e${timesExcavated.addSeparators()}",
+                listOf("§7You excavated §e${timesExcavated.addSeparators()} §7times.")
+            )
+        )
+
+        // TODO use same price source as profit tracker
+        profit = addScrap(timesExcavated, profit)
+        if (config.trackGlacitePowder) {
+            addGlacitePowder(data)
+        }
+
+        addAsSingletonList(tracker.addTotalProfit(profit, data.timesExcavated, "excarvation"))
+
+        tracker.addPriceFromButton(this)
+    }
+
+    private fun MutableList<List<Any>>.addGlacitePowder(data: Data) {
+        val glacitePowderGained = data.glacitePowderGained
+        if (glacitePowderGained <= 0) return
+        addAsSingletonList(
+            Renderable.hoverTips(
+                "§bGlacite Powder§7: §e${glacitePowderGained.addSeparators()}",
+                listOf(
+                    "§7No real profit,",
+                    "§7but still nice to see! Right?",
+                )
+            )
+        )
+    }
+
+    private fun MutableList<List<Any>>.addScrap(
+        timesExcavated: Long,
+        profit: Double,
+    ): Double {
+        if (timesExcavated <= 0) return profit
+        val scrapPrice = timesExcavated * scrapItem.getPrice()
+        addAsSingletonList(
+            Renderable.hoverTips(
+                "${scrapItem.itemName}§7: §c${NumberUtil.format(scrapPrice)}",
```

Comment:
make the price spent be shown as a negative number, to be consistent with the slayer profit tracker

```suggestion
                "${scrapItem.itemName}§7: §c-${NumberUtil.format(scrapPrice)}",
```